docs/contribute/redirects.md

```diff
@@ -16,6 +16,29 @@ For API redirects, consult with the documentation engineering team on Slack (#el
 
 For elastic.co/guide redirects, open a [web team request](http://ela.st/web-request).
 
+## Add a redirect
+
+To successfully implement a redirect:
+
+1. Locate and open the `redirects.yml` file to edit.
+   - Each docs repository powered by Docs V3 can have its own `redirects.yml` file. You must only edit the one in the repository that you're moving or removing files from.
+   - It is at the same location as the `docset.yml` file. Depending on the repository, this is generally at the root or within the `/docs` folder of the repository.
```

Comment:
very minor suggestion

I'd reorder this to put the info in the second bullet first, followed by `Each docs repository powered by Docs V3 can have its own redirects.yml file. .....` perhaps in a TIP